README.md

```diff
@@ -965,112 +1408,535 @@ export function trap(message: string): Query<boolean> {
+| stable write int32   | :x:                                                                                          | :heavy_check_mark:                                                                                                       | :x:                |
+| stable read int16    | :x:                                                                                          | :heavy_check_mark:                                                                                                       | :x:                |
+| stable write int16   | :x:                                                                                          | :heavy_check_mark:                                                                                                       | :x:                |
+| stable read int8     | :x:                                                                                          | :heavy_check_mark:                                                                                                       | :x:                |
+| stable write int8    | :x:                                                                                          | :heavy_check_mark:                                                                                                       | :x:                |
+| stable read float64  | :x:                                                                                          | :heavy_check_mark:                                                                                                       | :x:                |
+| stable write float64 | :x:                                                                                          | :heavy_check_mark:                                                                                                       | :x:                |
 
 ### Roadmap
 
--   [ ] 1.0
-    -   [ ] [Feature parity with Rust and Motoko CDKs](https://github.com/demergent-labs/azle/issues/134)
-    -   [ ] Core set of Azle-specific npm packages
-    -   [ ] Sudograph integration
-    -   [ ] Official dfx integration with `"type": "typescript"` or `"type": "azle"`
-    -   [ ] Live robust examples
-    -   [ ] Video series
-    -   [ ] Comprehensive benchmarks
-    -   [ ] Robust property-based tests
-    -   [ ] Optimized compilation
-    -   [ ] Security audits
--   [ ] 2.0
-    -   [ ] Azle VS Code plugin
-    -   [ ] [Inter-Canister Query Calls](https://forum.dfinity.org/t/inter-canister-query-calls-community-consideration/6754)
-
-### Limitations
+-   [ ] July 2022
+    -   [ ] Extensive automated benchmarking
+-   [ ] August 2022
+    -   [ ] Compiler error DX revamp
+    -   [ ] Rust rewrite
+-   [ ] September 2022
+    -   [ ] Extensive automated property testing
+    -   [ ] Multiple independent security reviews/audits
```

Comment:
I think we should remove all this and instead drop a link to our milestones: https://github.com/demergent-labs/azle/milestones. That is where we are actively setting our roadmap and we should be directing people there. That way we have the flexibility to edit the roadmap as needed rather than having to update the README every time we make a change.

Reply:
Correction, let's make that link https://github.com/demergent-labs/azle/milestones?direction=asc&sort=due_date&state=open so that it actually sorts properly for people.

Reply:
I want to give people a very high-level overview of what to expect, and we don't have these month-by-month milestones anymore. This is supposed to be a simple and concise high-level overview of what to expect. Maybe once we get our milestones actually sorted out and up and running again, they're not really working right now.